app/src/main/java/com/github/orkest/ui/feed/CommentActivity.kt

```diff
@@ -42,6 +42,31 @@ class CommentActivity : ComponentActivity() {
             }
         }
     }
+    companion object{
+    /**
+    * Comment design
+    * @param comment: the comment to display
+    */
+    @Composable
+    fun CommentBox(comment: Comment) {
+        Box(
+            modifier = Modifier
+                .fillMaxWidth()
+        ) {
+            Row {
+                displayProfilePic()
+                Column {
+                    Row(horizontalArrangement = Arrangement.SpaceEvenly){
+                        Text(text = comment.username, fontWeight = FontWeight.Bold, modifier = Modifier.testTag("comment_username"))
+                        Text(modifier = Modifier
+                            .padding(horizontal = 8.dp)
+                            .testTag("comment_date"), fontSize = 10.sp, text = comment.date.toString())
+                    }
+                    Text(text = comment.text, modifier = Modifier.testTag("comment_text"))
+                }
+            }
+        }
+    }}
```

Comment:
Maybe check that comment is not null.

Are you sure there will never be a formatting problem when doing comment.date.toString().

And finally why do you use a Box and not a Row as you're filling the entire width.